boards/arm/mimx8mq_evk/doc/index.rst

```diff
@@ -0,0 +1,184 @@
+
+The M4 can use up to 3 different RAMs. These are the memory mapping for A53 and M4:
+
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| Region     | Cortex-A53              | Cortex-M4 (System Bus) | Cortex-M4 (Code Bus)  | Size                 |
++============+=========================+========================+=======================+======================+
+| OCRAM      | 0x00900000-0x0091FFFF   | 0x20200000-0x2021FFFF  | 0x00900000-0x0091FFFF | 128KB                |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| TCMU       | 0x00800000-0x0081FFFF   | 0x20000000-0x2001FFFF  |                       | 128KB                |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| TCML       | 0x007E0000-0x007FFFFF   |                        | 0x1FFE0000-0x1FFFFFFF | 128KB                |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+| OCRAM_S    | 0x00180000-0x00187FFF   | 0x20180000-0x20187FFF  | 0x00180000-0x00187FFF | 32KB                 |
++------------+-------------------------+------------------------+-----------------------+----------------------+
+
+For more information about memory mapping see the
+`i.MX 8M Applications Processor Reference Manual`_  (section 2.1.2 and 2.1.3)
+
+At compilation time you have to choose which RAM will be used. This
+configuration is done in the file ``boards/arm/mimx8mq_evk/mimx8mq_evk.dts``
+with "zephyr,flash" (when CONFIG_XIP=y) and "zephyr,sram" properties.
+The available configurations are:
+
+.. code-block:: none
+
+   "zephyr,flash"
+   - &tcml_code
+   - &ocram_code
+   - &ocram_s_code
+
+   "zephyr,sram"
+   - &tcmu_sys
+   - &ocram_sys
+   - &ocram_s_sys
+
+Load and run Zephyr on M4 from A53 using u-boot by copying the compiled
+``zephyr.bin`` to the first FAT partition of the SD card and plug the SD
+card into the board. Power it up and stop the u-boot execution at prompt.
+
+Load the M4 binary onto the desired memory and start its execution using:
+
+.. code-block:: console
+
+   fatload mmc 0:1 0x40480000 zephyr.bin
+   cp.b 0x40480000 0x7e0000 0x8000
+   bootaux 0x7e0000
+
+Debugging
+=========
+
+MIMX8MQ EVK board can be debugged by connecting an external JLink
+JTAG debugger to the J401 debug connector and to the PC. Then
+the application can be debugged using the usual way.
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: mimx8mq_evk
+   :goals: debug
+
+Open a serial terminal, step through the application in your debugger, and you
+should see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS build zephyr-v1.13.0-30942-g6ee70bd22058 *****
```

Comment:
This is a really old Zephyr version, did you copy/paste this from another board page?

Reply:
At build time, the following warning occurs and it becomes `v1.13.0`. The documentation has been changed to `v2.6.99`.
```
-- Application: /work/zephyrproject/zephyr/samples/hello_world
-- git describe warned: warning: tag 'zephyr-v1.13.0' is really 'v1.13.0' here
-- Zephyr version: 2.6.99 (/work/zephyrproject/zephyr), build: zephyr-v1.13.0-32226-gb54dfe48c038
```